Makefile

```diff
@@ -49,6 +49,7 @@ test-ci:
 publish:
 	git pull --rebase
 	make build
+	make build-dist
```

Comment:
`build` is a part of `build-dist` so we're running it twice, so you can remove the line above this.